tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -999,7 +999,8 @@ def test_does_not_detect_consecutive_db_spans_with_truncated_query(self):
         assert sdk_span_mock.containing_transaction.set_tag.call_count == 0
 
     def test_detects_consecutive_db_spans(self):
-        span_duration = 5
+        SECOND = 1000
+        span_duration = 1 * SECOND
```

Comment:
Intentionally making this a second so that further threshold increases don't warrant a test update.